homeassistant/components/repairs/const.py

```diff
@@ -1,3 +1,6 @@
 """Constants for the Repairs integration."""
 
 DOMAIN = "repairs"
+
+INDEX_VIEW_URL = "/api/repairs/issues/fix"
+RESOURCE_VIEW_URL = "/api/repairs/issues/fix/{flow_id}"
```

Comment:
Now we see more and more repairs, I think it's nice if we could get some API for this like we have with config_entries

Reply:
Sure - but can we do that in a follow-up PR?

Reply:
Oh definetly, not here

Reply:
Refactored the PR to use helper methods